tests/laravel-test.sh

```diff
@@ -33,7 +33,7 @@ echo "Preparing Laravel"
 ./artisan make:test ExampleTest
 
 echo "Adding package from source"
-sed -e 's|"type": "project",|&"repositories": [ { "type": "path", "url": "../psalm-plugin-laravel" } ],|' -i composer.json
+sed -e 's|"type": "project",|&"repositories": [ { "type": "path", "url": "../psalm-plugin-laravel" } ],|' composer.json
```

Comment:
oh, this -i is needed I'm pretty sure. why'd ya remove it, mind explaining?

Reply:
The command didn't work in my shell, but I'll restore it. Probably something Mac related.

```
sed -e "s|'type': 'project',|&'repositories': [ { 'type': 'ath', 'url': '../psalm-plugin-laravel' } ],|" -i composer.json

sed: -I or -i may not be used with stdin
```